docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+  <MainSection name="What is the process to request new additions or changes?">
+    <MainSection.Subsection
+      description={`
+What we build into Gestalt comes from the teams across Pinterest, so we’d love to pair with you early and often! The process starts in the design phase. Seeing early mocks, wireframes or even product roadmaps in our [partnership meetings](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:h2=Meetings-and-Events) to learn where your project may be going helps inform our team OKRs. The earlier we see the work, the better we’re able to plan and the higher the likelihood we’ll be able to help.
+We ask that any ideas that change the functionality of an existing Gestalt component or introduce a net-new component go through the following process:
+1. **Present work** **early** **in our** [**partnership meetings**](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:h2=Meetings-and-Events)
+    - Explain project goals and show multiple options explored with and without Gestalt components to help us understand your needs. This does not need to be a polished presentation, but should have enough detail that we understand the ask and why it is needed.
+    - Coordinate with other designers if there are overlapping needs to help us better prioritize in our roadmap.
+    - Be able to explain how this component should be used or not used. This helps us in testing the component against existing components and product surfaces.
+    - Have a rough project timeline available. This allows us to determine if we’re able to accommodate the work.
+2. **Iterate on solutions**
+    We love to see more iteration from the product designer asking for the component, if they have the time. If not, the component will follow our prioritization process.
+    After an initial crit, the designer (product designer or Gestalt designer) must ensure all edge cases have been considered and come back to component crit as needed.
+    Some things to finalize before we can build the component:
+    - Empty states
+    - Focus states
+    - Truncation/internationalization options
+    - Responsive behavior
+    - Dark mode support
+3. **Prioritization by the Gestalt team**
+    If our team takes on the work, we will add it to our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100) and prioritize it appropriately. Typically the determining factor for taking on work is capacity: if your designers or engineers have the capacity, we’d love for them to [contribute to Gestalt](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:uid=384006891589287519571583&h2=How-can-you-contribute-to-Gest) with our support. Otherwise, the Gestalt designers and engineers will prioritize the work against our current workload based on the following criteria.
```

Comment:
this should be` [contribute to Gestalt](#How-can-you-contribute-to-gestalt-as-an-engineer)`

Reply:
Done